doc/quick-start-guide-and-build.md

```diff
@@ -131,7 +131,9 @@ You can now create a new pull request or update an existing one with commits. _L
 
 Before building _LPVS_ from source code, ensure that you have the following prerequisites installed:
 
-- SCANOSS Python package by following the [guidelines](https://github.com/scanoss/scanoss.py#installation). Install it using the command:
+- SCANOSS Python package by following the [guidelines](https://github.com/scanoss/scanoss.py#installation).
+  
+  For installing SCANOSS on Ubuntu 2023.04, Fedora 38, Debian 11, etc. or later versions of mentioned OSes please follow the note at the end of the paragraph otherwise install it using the command:
```

Comment:
Please do not use such abbreviations.

Reply:
Please clarify -is it related to name of OSes? I do not know how to inform an user if needed special steps if no information of OS name and version.

Reply:
I mention the abbreviation `OS` here.

Reply:
Thanks for comment. Is there a variant? And how to inform user that needed to use pipx instead pip if no OS reference? May be Python version? But on scanoss guide they referenced to name of OSes and versions.

Reply:
My suggestion is the following.

Could you try to install ScanOSS using such a command? In case of failure, please refer to the official documentation of ScanOSS.